### PR TITLE
Ft-heroku-deploy-154956015

### PR DESCRIPTION
**Pull request by Kalela**
### What does this PR do?
- Deploys the Health Checks app to Heroku.
- Edits the README file with updated information on the project.
### How should this be manually tested?
- Run the Heroku app through the link provided in the README file.
- Follow instructions on the README file to use HealthChecks.

### What are the relevant pivotal tracker stories?
Deploy app to heroku.
ID: #154956015

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 healthchecks is a watchdog for your cron jobs. It's a web server that listens for pings from your cron jobs, plus a web interface.
```
Comment (meshnesh) [inline README.md:16]: Good cool idea noting this change, although going through the whole file there are a lot of things that can be changed.

File: .travis.yml [hunk 2]
```diff
@@ -1,16 +1,13 @@
 language: python
 python:
   - "3.6"
-  - "2.7"
   - "3.4"
```
Comment (meshnesh) [inline .travis.yml]: I think you should also remove the version `3.4` since most of us are using `3.6`

File: README.md (hunk 1 above)
Comment (betsybeth) [reply README.md]: Great work.The title of PR,I think it will be better if you include the pivotal tracker ID also

File: .travis.yml (hunk 2 above)
Comment (betsybeth) [reply .travis.yml]: Great work.The title of PR,I think it will be better if you include the pivotal tracker ID also

Comment (FelixWambiri) [reply .travis.yml]: Yeah it is a pretty good idea to change to the most used version amongst us

**Comment by betsybeth** [comment]
Great work.The title of PR,I think it will be better if you include the pivotal tracker ID also

**Comment by mutoromike** [comment]
Great method of PR description and fixing the failing tests. I will start implementing that on mine too.

File: .travis.yml (hunk 2 above)
Comment (Kalela) [reply .travis.yml]: sure thing. I feel it doesn't hurt to have it though.

**Review by lakivisi-zz — COMMENTED** [review]
The staticfiles were generated, right? @Kalela

**Comment by Kalela** [comment]
Yeah. I ran a command to generate the files. python manage.py collectstatic.

File: .travis.yml (hunk 2 above)
Comment (meshnesh) [reply .travis.yml]: it's going to avoid travis running the environment for that version

**Comment by lakivisi-zz** [comment]
Instead of committing the static files; add the command to travis.yml so that whenever there is a build, it collects static leaving our repo clean

**Comment by Kalela** [comment]
Lemme do the changes now

**Comment by lakivisi-zz** [comment]
Remember to add staticfiles to gitignore

**Pull request merged**